### ISSUE TITLE
NTR, pulmo: Nodular-centrilobular without tree-in-bud pattern

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5574]
**Preferred term label:**
Nodular-centrilobular without tree-in-bud pattern

**Synonyms**

**Definition (free text, please give PubMed ID)**
A nodular pattern on pulmonary high-resolution computed tomography which are anatomically located centrally within secondary pulmonary lobules. Centrilobular nodules may be dense (i.e., solid) and of homogeneous opacity or ground-glass opacity, and may range from a few millimeters to about 1 cm in size. Because of the similar size of secondary lobules, centrilobular nodules often appear to be evenly spaced. Centrilobular nodules are usually separated from the pleural surfaces, fissures, and interlobular 
septa by a distance of at least several millimeters. They may appear patchy or diffuse in different diseases

**Parent term (use hpo.jax.org/app)**
Nodular pattern on pulmonary HRCT HP:0025392

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
associated to bronchiolar or vascular disease: Bronchiolitis obliterans with obstructive pulmonary disease (ORPHA: 1303), Mycobacterium Tuberculosis (OMIM 607948, ORPHA:3389), Endobronchial Mycobacterium avium Infection, Bronchopneumonia, Bronchoalveolar Carcinoma with airway spread (BAC), Hypersensitivity Pneumonitis (OMIM 145300, ORPHA:31740), Respiratory Bronchiolitis (ORPHA:79127), Cryptogenic organizing Pnemonia (BOOP) (ORPHA:1302), Pneumoconiosis (ORPHA:182098), Histocytosis, Pulmonary Edema (ORPHA:330012), pulmonary vasculitides (f.e. ORPHA:183 Eosinophilic granulomatosis with polyangiitis). 

**Your nano-attribution (ORCID)**
LMU crapp, @mgriese, @katrinknoflach

**Comment by pnrobinson** [comment]
adding term as HP:0032972
